libs/features/icons/icons.js

```diff
@@ -46,6 +46,10 @@ function decorateToolTip(icon) {
   // Position is the next to last part of a tooltip
   const place = conf.pop()?.trim().toLowerCase() || 'right';
   icon.className = `icon icon-info milo-tooltip ${place}`;
+  icon.setAttribute('tabindex', '0');
+  icon.setAttribute('role', 'button');
+  icon.setAttribute('aria-label', content);
+  icon.setAttribute('aria-describedby', content);
```

Comment:
I'm not sure this is needed. `aria-describedby` should reference an ID, documentation [here](https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Attributes/aria-describedby). Setting the `aria-label` should probably be enough.